rs-permission-converter/test/permconv.test.js

```diff
@@ -0,0 +1,67 @@
+const os = require('os');
+const path = require('path');
+const fs = require('fs');
+const fsExtra = require('fs-extra');
+const exec = require('@actions/exec');
+
+// Used by @actions/tool-cache. Directory must exist before we load module.
+const outputDir = path.join(__dirname, '..', 'test_output_dir');
+
+if (!process.env.NEXUS_USERNAME) {
+  // eslint-disable-next-line no-console
+  console.log('Using local user credentials in test');
+  process.env.NEXUS_USERNAME = process.env.GITHUB_USER;
+  process.env.NEXUS_PASSWORD = process.env.GITHUB_TOKEN;
+}
+
+// Directory must exist for tool-cache.
+fsExtra.mkdirs(outputDir);
+
+// We must import pkgbuilder AFTER preparing the environment.
+const { convertPermissions, downloadPermConvTool } = require('../src/permconv');
+
+jest.setTimeout(30000);
+
+describe('RS Permission Converter Tests', () => {
+  beforeAll(() => {
+    fsExtra.mkdirs(outputDir);
+  });
+
+  afterAll(() => {
+    fsExtra.removeSync(outputDir);
+  });
+
+  test('It downloads the build tool', async () => {
+    const tool = await downloadPermConvTool({
```

Comment:
I don't think we shall test the actual download. We need to introduce a unit/integration test for the loadTool method in utils, but in these actions it should be sufficient to mock this to speed up the tests.